spec.html

```diff
@@ -31144,7 +31144,7 @@ <h1>Pattern</h1>
           1. Return a new Abstract Closure with parameters (_str_, _index_) that captures _m_ and performs the following steps when called:
             1. Assert: Type(_str_) is String.
             1. Assert: ! IsNonNegativeInteger(_index_) is *true* and _index_ &le; the length of _str_.
-            1. If _Unicode_ is *true*, let _Input_ be a List consisting of the sequence of code points of ! StringToCodePoints(_str_). Otherwise, let _Input_ be a List consisting of the sequence of code units that are the elements of _str_. _Input_ will be used throughout the algorithms in <emu-xref href="#sec-pattern-semantics"></emu-xref>. Each element of _Input_ is considered to be a character.
+            1. If _Unicode_ is *true*, let _Input_ be ! StringToCodePoints(_str_). Otherwise, let _Input_ be a List containing the code units that are the elements of _str_. _Input_ will be used throughout the algorithms in <emu-xref href="#sec-pattern-semantics"></emu-xref>. Each element of _Input_ is considered to be a character.
```

Comment:
In some places we specify "in order"; it's not clear why this case should be different.

Reply:
Like this?

```suggestion
            1. If _Unicode_ is *true*, let _Input_ be ! StringToCodePoints(_str_). Otherwise, let _Input_ be a List containing the code units that are the elements of _str_ in order. _Input_ will be used throughout the algorithms in <emu-xref href="#sec-pattern-semantics"></emu-xref>. Each element of _Input_ is considered to be a character.
```

Reply:
Yeah. Similarly for three other cases below, I guess.

Reply:
Or we could go the other way and decide the order is implicit in all these cases; I don't much care which as long as we're consistent.

Reply:
I think it's technically ambiguous without it, but you'd have to be a real maniac to interpret the text any other way. I'm neutral. @tc39/ecma262-editors Third opinion?

Reply:
I think the order is implicit in all cases, and if it's not implicit, then "in order" also is not explicit _enough_ - in which order? alphabetical? insertion? reverse? random?

It seems simpler (and less needlessly pedantic) to me to just omit it everywhere.

Reply:
If there are places in the spec where the lack of "in order" was meant to signal "in no specified order", then it would be a normative change to specify that lack of "in order" means "in List order".

Reply:
@jmdyck I don't think anyone believes it was meant by the spec authors to signal "in no specified order"; the point of being explicit would just be to avoid potential misinterpretation by readers of the spec.

Reply:
@bakkot I removed all applicable occurrences of "in order".

Reply:
> I don't think anyone believes it was meant by the spec authors to signal "in no specified order"

That's good then.